### ISSUE TITLE
Value is undefined, expected an object

### ISSUE THREAD

**Issue by mlecoq** [mrousavy/react-native-blob-jsi-helper#6]
Hi, 

during my development I have a strange issue after hot reloading (on Android - with hermes) on calling `getArrayBufferForBlob`:

`Value is undefined, expected an object`

I have to restart metro after any code change

**Comment by mlecoq** [comment]
To precise, it appears not on every reload, just when there is an issue in last edited code

**Comment by mlecoq** [comment]
I close it, I am no more able to reproduce it

**Comment by ArkanRomanPango** [comment]
@mlecoq I also reproduce it. 
![image](https://user-images.githubusercontent.com/61008762/189875536-709c807c-1380-4436-bbae-40f2b83f5fbf.png)

**Comment by alexstanbury** [comment]
I'm seeing this problem too, so far only tested on iOS 16.4 Simulator. If code is hot reloaded, I get this error until I force quit the app and reopen, at which point all is good in the world until the next reload. Makes using this library while developing pretty painful.

Here's my code:
```  
const blob = await fetch(url).then(r => r.blob());
const arrayBuffer = getArrayBufferForBlob(blob);
// error here
```

@mrousavy can this be reopened please?

**Comment by mlecoq** [comment]
@alexstanbury waiting for your repro !

**Comment by hannojg** [comment]
Love the reproduction steps 😄  thank you so much for providing a reproduction 😊

**Comment by mrousavy** [comment]
Thanks for the repro! Can you throw a breakpoint there in Xcode and step through one by one and let me know in which line it fails?

https://github.com/mrousavy/react-native-blob-jsi-helper/blob/1d016b62e43ce44b4c5d0a810335567d9992e023/ios/BlobJsiHelper.mm#L36

**Comment by alexstanbury** [comment]
Hi Marc, it fails on this line:

https://github.com/mrousavy/react-native-blob-jsi-helper/blob/1d016b62e43ce44b4c5d0a810335567d9992e023/ios/BlobJsiHelper.mm#L44

**Comment by mrousavy** [comment]
Ah hold on I got a fix

**Comment by mrousavy** [comment]
Done, released in [v0.3.1](https://github.com/mrousavy/react-native-blob-jsi-helper/releases/tag/v0.3.1)!

**Comment by alexstanbury** [comment]
Confirming fix works, thank you! 🍻
